db/fixtures/miq_event_definitions.csv

```diff
@@ -106,7 +106,7 @@ vm_provisioned,VM Provision Complete,Default,vm_process
 vm_start,VM Power On,Default,vm_operational
 request_vm_start,VM Power On Request,Default,vm_operational
 # vm_shutdown,VM Shutdown,Default,vm_operational
-# vm_resume,VM Resume,Default,vm_operational
+vm_resume,VM Resume,Default,vm_operational
```

Comment:
@gmcculloug This one can be merged if there is no other update. And we are going to add the vm_resume for VMWare and RHEV in a separate pr.